ramls/examples/errors_500.sample

```diff
@@ -0,0 +1,8 @@
+{
```

Comment:
I don't see the case where we can use this sample in scope of this module. I think raml/examples folder should be deleted at all as it contains not required samples.